pom.xml

```diff
@@ -32,7 +32,7 @@
         <gson.version>2.8.9</gson.version>
         <guava.version>30.1.1-jre</guava.version>
```

Comment:
It looks like the 11/17/2021 patch is now using this Guava version. If that's the case, then this variable and the Guava-related import(s) should be removed. You could also remove the WAR-filtering-based and KFS-dependency-based exclusions of Guava.

Reply:
updated